mitodl/open-discussions
# OLX ingestion for xPRO courses

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #2575 

#### What's this PR do?
Adds support for ingesting OLX courses from S3, using a similar (or same if applicable) pipeline as the OCW course ingestion.

#### How should this be manually tested?
- Set the necessary environment variables (contact me for these values)
 - Run `./manage.py backpopulate_xpro_data` to create courses and runs
 - Then run `./manage.py backpopulate_xpro_files` to download the course OLX data. This will be used to create `ContentFile` instances which will be included

Finally, go to `/learn/search`, filter on xPRO courses and search for thermoplastic. You should see only one result named "Additive Manufacturing for Innovative Design and Production".
